css/properties/clip-rule.json

```
@@ -10,12 +10,12 @@
           ],
           "support": {
             "chrome": {
-              "version_added": "≤80"
+              "version_added": "≤15"
             },
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": "≤72"
+              "version_added": "≤4"
```

I think it was implemented in https://bugzilla.mozilla.org/show_bug.cgi?id=160403 / https://github.com/mozilla/gecko-dev/commit/0ee1400807c24990945293384b4991d9a6d9cea0#diff-0902cca9b3a69307fb9842630ce3a29de20083d0f964efd493078de7dc7a197a, in Gecko 1.9.1, i.e. Firefox 3.5.

```suggestion
              "version_added": "3.5"
```